open-machine-learning-jupyter-book/assignments/python-programming-basics.ipynb

```diff
@@ -1,59 +1,502 @@
-{

```

Comment:
<p>https://towardsdatascience.com/write-markdown-latex-in-the-jupyter-notebook-10985edb91fd</p><p> </p><p>Follow this to update the math formulation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>